test/instrumentation/modules/ioredis.js

```diff
@@ -55,7 +56,7 @@ test('not nested', function (t) {
 test('nested', function (t) {
   resetAgent(done(t))
 
-  var redis = new Redis()
+  var redis = new Redis('6379', host)
```

Comment:
You don't need to specify the default port. If the host is the first argument it will work fine as well 😃

If you simply inline the environment variable, then it will default to `localhost` if it's not set. I.e:

```js
new Redis(process.env.REDIS_HOST)
```